libs/application/templates/parental-leave/src/lib/parentalLeaveUtils.ts

```diff
@@ -254,6 +254,30 @@
   return days
 }
 
+export const getPersonaldays = (application: Application) => {
+  const selectedChild = getSelectedChild(
+    application.answers,
+    application.externalData,
+  )
+  if (!selectedChild) {
+    throw new Error('Missing selected child')
+  }
+  const maximumDaysToSpend = getAvailableRightsInDays(application)
+  const maximumMultipleBirthsDaysToSpend = getMultipleBirthsDays(application)
+  const maximumAdditionalSingleParentDaysToSpend =
+    getAdditionalSingleParentRightsInDays(application)
+  const transferredDays = getTransferredDays(application, selectedChild)
+  const personalDays =
+    maximumDaysToSpend -
+    maximumAdditionalSingleParentDaysToSpend -
+    maximumMultipleBirthsDaysToSpend -
+    Math.max(transferredDays, 0)
```

Comment:
Math.max is needed since transferred days can be a negative value and we want to only subtract if transferred days is positive. The reason for this is because personal days should not be more than the basic rights.